Configuration/odin-app.json

```diff
@@ -17,6 +17,22 @@
                     }
                 }
             }
+        },
+        "views/odin/app/secure": {
```

Comment:
I don't think we need this in this config file. The view doesn't really do anything does it? If you need this for a test it should go in a test view JSON config file.